security.rst

```diff
@@ -300,8 +302,20 @@ you are anonymous:
 .. image:: /_images/security/anonymous_wdt.png
    :align: center
 
+It will also allow a request to access a form login without being authenticated as a
+unique user (otherwise an infinite redirection loop would happen asking the user to
+authenticate while trying to doing so).
 You'll learn later how to deny access to certain URLs or controllers.
 
+.. note::
+
+    The "lazy" anonymous mode prevent the session from being started if there is
+    no need for authorization (i.e explicit check for a user privilege).
```

Comment:
What about adding the usefullness of this, e.g. with "This makes those requests cachable.".

Reply:
Good call, done.